Sources/App/Extensions/Router+Extensions.swift

```diff
@@ -10,23 +10,19 @@ import Vapor
 extension Request {
     var acceptLanguage: String {
         get {
-            guard let accpetLanguage = self.http
+            let accpetLanguage = String(self.http
```

Comment:
Maybe `acceptLanguage` instead of `accpetLanguage`

Reply:
Fixed